web/massastation/src/pages/Store/StationSection/StationPlugin.tsx

```diff
@@ -81,19 +83,31 @@ export function StationPlugin({
     version: version,
   } as PluginProps;
 
+  // Temporary display of beta tag for this specific plugin
+  // https://github.com/massalabs/station/issues/1206
+  let defaultContent =
+    name === 'Massa Node-manager'
+      ? [<Tag type="warning" content="beta" />]
+      : [];
```

Comment:
This is weird... please, keep an issue open to track that. Otherwise we'll forget!

Reply:
Ya, it's a quick win, we will take followup issue before putting the task in Done